### ISSUE TITLE
Edit CotC Signature

### ISSUE THREAD

**Issue by cholly75** [ustaxcourt/ef-cms#8566]
As the Chief Judge, so that the Clerk of the Court is properly represented on official documentation, I need to be able to easily modify the Clerk of the Court signature added the bottom of various notices and documentation.

In January 2024 a new Clerk of the Court will be appointed, potentially on an interim basis. As the CotC's current name and title are automatically added to a number of documents (primarily notices) in DAWSON automatically, we need to be sure we can modify this name and title of the officeholder as needed and test the system for any adjustments needed re: placement, sizing, or positioning on the documentation.

The name is of course subject to change based on whoever gets hired, either temporarily or permanently. The title may need to be modified to put "Interim" or "Acting" in front of "Clerk of the Court", depending on the decision of the Court. We would like to modify this value in pre-production ahead of time to discover any additional work that may need to be done prior to January 2024.

## Acceptance Criteria
* Name of CotC can be modified to display everywhere we currently display the CotC signature
* Title of CotC can be modified to display everywhere we currently display the CotC signature

## Notes
* Any sizing/placement issues arising from the new values from name/title will be addressed in the context of stories about those particular instances if necessary

## Story Definition of Ready (updated on 12/23/22)
The following criteria must be met in order for the user story to be picked up by the Flexion development team.
The user story must: 
- [ ] Is framed in business/user need, the value has been addressed.
- [ ] Includes acceptance criteria
- [ ] Has been refined
- [ ] Pre conditions have been satisfied. 

**Process:** 
Flexion developers and designers will test if the story meets acceptance criteria and test cases in Flexion dev and staging environments (“standard testing”). If additional acceptance criteria or testing scenarios are discovered while the story is in progress, a new story should be created, added to the backlog and prioritized by the product owner. 

## Definition of Done (Updated 5-19-22)
**Product Owner**
 - [ ]  Acceptance criteria have been met and validated on the Court's migration environment
 - [ ] Add scenario to testing document, if applicable (https://docs.google.com/spreadsheets/d/1FUHKC_YrT-PosaWD5gRVmsDzI1HS_U-8CyMIb-qX9EA/edit?usp=sharing)

**UX**
 - [ ] Business test scenarios have been refined to meet all acceptance criteria 
 - [ ] Usability has been validated
 - [ ] Wiki has been updated (if applicable) 
 - [ ] Story has been tested on a mobile device (for external users only)

**Engineering**
 - [x] Automated test scripts have been written, including visual tests for newly added PDFs.
 - [x] Field level and page level validation errors (front-end and server-side) integrated and functioning.
 - [x] Verify that language for docket record for internal users and external users is identical.
 - [x] New screens have been added to pa11y scripts.
 - [x] All new functionality verified to work with keyboard and macOS voiceover https://www.apple.com/voiceover/info/guide/_1124.html.
 - [x] READMEs, other appropriate docs, and swagger/APIs fully updated.
 - [x] UI should be touch optimized and responsive for external only (functions on supported mobile devices and optimized for screen sizes as required).
 - [x] Interactors should validate entities before calling persistence methods.
 - [ ] Code refactored for clarity and to remove any known technical debt.
 - [x] If new docket entries have been added as seed data to `efcms-local.json`, 3 local s3 files corresponding to that docketEntryId have been added to `web-api/storage/fixtures/s3/noop-documents-local-us-east-1`
 - [x] Acceptance criteria for the story has been met.
 - [x] If there are special instructions in order to deploy into the next environment, add them as a comment in the story.
 - [x] If the work completed for the story requires a reindex without a migration, or any other special deploy steps, apply these changes to the following flexion branches:
   - [x] experimental1
   - [x] experimental2
   - [x] experimental3
   - [x] experimental4
   - [x] experimental5
   - [x] experimental6
   - [x] develop
 - [x] Reviewed by UX on a deployed environment.
 - [ ] Reviewed by PO on a deployed environment. Can be deployed to the Court's `test` environment if prod-like data is required. Otherwise deployed to any `experimental` environment.
 - [ ] Deployed to the Court's `staging` environment.

**Comment by swongCO** [comment]
Pre refinement
- does the Chief Judge need to enter the signature or tell us what the name/title should be? no need for judge to enter it; no need for a UI in Dawson to do this
- what kind of latency do we need for this update? i.e. tied to a deploy or happen in real time - Chris is agnostic on this as long as the court can test this

**Comment by ttlenard** [comment]
## Test Cases

#### 1) DAWSON technical team member updates the Clerk of the Court's Name; Leaves title as is; Notices in DAWSON display the updated CotC name.

- As a Technical Team Member, update the CotC name (Dev Determination on how to do this)

**Part 1 - NOTR/Notice of attachments - eFiled Petition**
- Next, Log into DAWSON as a Petitions Clerk
- Click on Document QC
- Find a Petition that is in New status (avoid the test petitions that are in a status other than new)
- Click on the Petition link
- Be sure that you click on the radio button for Notice of Attachments in the Nature of Evidence while you are QCing the eFiled Petition.
- Serve the Petition to the IRS
- Click on the Served tab
- Click on the docket number of the Petition that you just served.
- Click on the Notice of Receipt of Petition on the docket record

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court name at the bottom of the page displays the updated CotC name, title is Clerk of the Court as per normal.

- Next, click on the Case information tab
- Click on the Print Confirmation link

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the page displays the updated CotC name, title is Clerk of the Court as per normal.

- Next, click on the drafts tab and find the Notice of Attachments in the Nature of Evidence draft

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Draft Notice displays the updated CotC name, title is Clerk of the Court as per normal
- [x] **NEW FUNCTIONALITY** - After clicking on the Add Docket Entry link the document preview on the Add Docket Entry screen has the updated CotC name, title is Clerk of the Court as per normal
- [x] **NEW FUNCTIONALITY** - After Serving the document, the document on the docket record displays the updated CotC name, title is Clerk of the Court as per normal.



**Part 2 - NOTR/Notice of attachments - Paper-Filed Petition**
- Log into DAWSON as a Petitions Clerk
- Click on Document QC
- Click on Start a Petition button
- Upload or scan in documents and fill in all the required fields
- Be sure that you click on the radio button for Notice of Attachments in the Nature of Evidence while you are QCing the eFiled Petition.
- Create the Case, and then Serve the Petition to the IRS

Expected Results
- [x] **NEW FUNCTIONALITY** - On the Print Paper Petition Receipt screen, The Clerk of the Court name at the bottom of the NOTR displays the updated CotC name, title is Clerk of the Court as per normal.  
- [x] If the document is printed, the Petitioner address fits in the envelope window as per normal.

- Next, click the done button
- Click on the Served link in your Document QC
- Click on the docket number link of the petition you just served
- Click on the Notice of Receipt of Petition on the docket record

Expected Results:
- [x] Notice of receipt of petition (Check to be sure that the address window still looks good)
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court name at the bottom of the page displays the updated CotC name, title is Clerk of the Court as per normal.

- Next, click on the Case information tab
- Click on the Print Confirmation link

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the page displays the updated CotC name, title is Clerk of the Court as per normal.

- Next, click on the drafts tab and find the Notice of Attachments in the Nature of Evidence draft

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Draft Notice displays the updated CotC name, title is Clerk of the Court as per normal
- [x] **NEW FUNCTIONALITY** - After clicking on the Add Docket Entry link the document preview on the Add Docket Entry screen has the updated CotC name, title is Clerk of the Court as per normal
- [x] **NEW FUNCTIONALITY** - After Serving the document, the document on the docket record displays the updated CotC name, title is Clerk of the Court as per normal.



**Part 3 - Generate a Notice on a case**

- Continuing from part 2 above
- Click on the Create button in the case
- Select Order or Notice
- Select Notice and then click Continue
- Type in some text in the Generate section on the Create Notice screen
- Click on the Preview tab

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the page displays the updated CotC name, title is Clerk of the Court as per normal.

- Next, Click on the Save button

Expected Results:
- [x] **NEW FUNCTIONALITY** - The document preview on the Drafts page displays The Clerk of the Court signature at the bottom of the page displays the updated CotC name, title is Clerk of the Court as per normal.

- Next, click on the Add docket Entry link

Expected Results:
- [x] **NEW FUNCTIONALITY** - The document preview on the Add Docket Entry page displays The Clerk of the Court signature at the bottom of the page displays the updated CotC name, title is Clerk of the Court as per normal.

- Next, Save and Serve the document
- Review the Print Notice for Paper Service Screen

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the page displays the updated CotC name, title is Clerk of the Court as per normal.



**Part 4 - Notice of Docket Change**

- Log in as a party that has electronic access to a case
- Click on the docket number for the case on the docket record
- Click on File a document button
- Select a document title, upload a document, and submit the document to the Court
- Next, log in as a docket clerk
- Click on Document QC, and then Section Document QC 
- Click on the document title link for the document that you just filed
- Change the document title and click Complete
- Navigate back to the case docket record
- Click on the automatically generated Notice of Docket Change

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the page displays the updated CotC name, title is Clerk of the Court as per normal.



**Part 5 - Initial Calendaring**

- Log in as a Petitions clerk (or CotC, or CSS)
- Click on Trial Sessions
- Click on Add Trial Session button
- Fill in the required fields for the trial session
- Click Add Session
- Next, click on the New tab and then click on the City, State link for the session you just created
- Select a few cases from the Eligible case list, making sure that you select at least 1 case that has petitioners with paper service.
- Click Set Calendar
- Review the Print for paper service screen

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the Notice setting case for Trial for each case displays the updated CotC name, title is Clerk of the Court as per normal.
- [x] **NEW FUNCTIONALITY** - After clicking done and opening some of the cases on the trial session, Click on the Notice that was just generated on the docket record - The Clerk of the Court signature at the bottom of the Notice setting case for Trial is displayed correctly, title is Clerk of the Court as per normal.



**Part 6 - Notice of Change to Remote Proceeding**

- Continuing from the previous Test
- Click on trial Sessions
- Click on the city, state link for the trail session from the previous test
- Click on the edit link
- Select the remote radio button, and fill in the required fields for a remote proceeding
- Save

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the Notice of Change to Remote Proceeding for each case displays the updated CotC name, title is Clerk of the Court as per normal.
- [x] **NEW FUNCTIONALITY** - After clicking done and opening some of the cases on the trial session, Click on the Notice that was just generated on the docket record - The Clerk of the Court signature at the bottom of the Notice of remote proceeding is displayed correctly, title is Clerk of the Court as per normal.



**Part 7 - Notice of Change to In Person Proceeding**

- Continuing from the previous Test
- Click on trial Sessions
- Click on the city, state link for the trail session from the previous test
- Click on the edit link
- Select the In-Person radio button, and fill in the required fields for an In-Person proceeding
- Save

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the Notice of Change to In-Person Proceeding for each case displays the updated CotC name, title is Clerk of the Court as per normal.
- [x] **NEW FUNCTIONALITY** - After clicking done and opening some of the cases on the trial session, Click on the Notice that was just generated on the docket record - The Clerk of the Court signature at the bottom of the Notice of In-Person Proceeding is displayed correctly, title is Clerk of the Court as per normal.



**Part 8 - Notice of Change of Trial Judge**

- Continuing from the previous Test
- Click on trial Sessions
- Click on the city, state link for the trail session from the previous test
- Click on the edit link
- Scroll down to the Judge Dropdown, and select a different Judge.  Be sure all required fields are filled out.
- Save

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the Notice of Change of Trial Judge for each case displays the updated CotC name, title is Clerk of the Court as per normal.
- [x] **NEW FUNCTIONALITY** - After clicking done and opening some of the cases on the trial session, Click on the Notice that was just generated on the docket record - The Clerk of the Court signature at the bottom of the Notice of change of trial judge is displayed correctly, title is Clerk of the Court as per normal.



**Part 9 - 30 Day Notice of Trial**

- Log in as a Petitions Clerk (or CSS, or CotC)
- Click on Trial Sessions 
- Scroll down until you find a trial session that has a clock icon present
- Click on the City, state link for the trial session with a clock icon
- Click on the Serve/Dismiss link in the yellow alert banner
- Click Yes, Serve button in the pop-up.
- Review the paper service page

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the 30 Day Notice of Trial for each case displays the updated CotC name, title is Clerk of the Court as per normal.
- [x] **NEW FUNCTIONALITY** - After clicking Printing Complete and opening some of the cases on the trial session, Click on the Notice that was just generated on the docket record - The Clerk of the Court signature at the bottom of the 30 Day Notice of Trial is displayed correctly, title is Clerk of the Court as per normal.



#### 2) DAWSON Technical Team member updates the Clerk of the Court's Title, Leaves Name as is; Notices in DAWSON display the updated CotC title.

- As a Technical Team Member, update the CotC the title only (put "acting Clerk of the Court") (Dev Determination on how to do this)

**Part 1 - NOTR/Notice of attachments - eFiled Petition**
- Next, Log into DAWSON as a Petitions Clerk
- Click on Document QC
- Find a Petition that is in New status (avoid the test petitions that are in a status other than new)
- Click on the Petition link
- Be sure that you click on the radio button for Notice of Attachments in the Nature of Evidence while you are QCing the eFiled Petition.
- Serve the Petition to the IRS
- Click on the Served tab
- Click on the docket number of the Petition that you just served.
- Click on the Notice of Receipt of Petition on the docket record

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court name at the bottom of the page displays the CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".

- Next, click on the Case information tab
- Click on the Print Confirmation link

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the page displays the CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".

- Next, click on the drafts tab and find the Notice of Attachments in the Nature of Evidence draft

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Draft Notice displays the CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".
- [x] **NEW FUNCTIONALITY** - After clicking on the Add Docket Entry link the document preview on the Add Docket Entry screen has the CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".
- [x] **NEW FUNCTIONALITY** - After Serving the document, the document on the docket record displays the CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".



**Part 2 - NOTR/Notice of attachments - Paper-Filed Petition**
- Log into DAWSON as a Petitions Clerk
- Click on Document QC
- Click on Start a Petition button
- Upload or scan in documents and fill in all the required fields
- Be sure that you click on the radio button for Notice of Attachments in the Nature of Evidence while you are QCing the eFiled Petition.
- Create the Case, and then Serve the Petition to the IRS

Expected Results
- [x] **NEW FUNCTIONALITY** - On the Print Paper Petition Receipt screen, The Clerk of the Court name at the bottom of the NOTR displays the CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".  
- [x] If the document is printed, the Petitioner address fits in the envelope window as per normal.

- Next, click the done button
- Click on the Served link in your Document QC
- Click on the docket number link of the petition you just served
- Click on the Notice of Receipt of Petition on the docket record

Expected Results:
- [x] Notice of receipt of petition (Check to be sure that the address window still looks good)
- [x] **NEW FUCTIONALITY** - The Clerk of the Court name at the bottom of the page displays the CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".

- Next, click on the Case information tab
- Click on the Print Confirmation link

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the page displays the CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".

- Next, click on the drafts tab and find the Notice of Attachments in the Nature of Evidence draft

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Draft Notice displays the CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".
- [x] **NEW FUNCTIONALITY** - After clicking on the Add Docket Entry link the document preview on the Add Docket Entry screen has the CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".
- [x] **NEW FUNCTIONALITY** - After Serving the document, the document on the docket record displays the CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".



**Part 3 - Generate a Notice on a case**

- Continuing from part 2 above
- Click on the Create button in the case
- Select Order or Notice
- Select Notice and then click Continue
- Type in some text in the Generate section on the Create Notice screen
- Click on the Preview tab

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the page displays the CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".

- Next, Click on the Save button

Expected Results:
- [x] **NEW FUNCTIONALITY** - The document preview on the Drafts page displays The Clerk of the Court signature at the bottom of the page displays the CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".

- Next, click on the Add docket Entry link

Expected Results:
- [x] **NEW FUNCTIONALITY** - The document preview on the Add Docket Entry page displays The Clerk of the Court signature at the bottom of the page displays the CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".

- Next, Save and Serve the document
- Review the Print Notice for Paper Service Screen

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the page displays the CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".



**Part 4 - Notice of Docket Change**

- Log in as a party that has electronic access to a case
- Click on the docket number for the case on the docket record
- Click on File a document button
- Select a document title, upload a document, and submit the document to the Court
- Next, log in as a docket clerk
- Click on Document QC, and then Section Document QC 
- Click on the document title link for the document that you just filed
- Change the document title and click Complete
- Navigate back to the case docket record
- Click on the automatically generated Notice of Docket Change

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the page displays the CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".



**Part 5 - Initial Calendaring**

- Log in as a Petitions clerk (or CotC, or CSS)
- Click on Trial Sessions
- Click on Add Trial Session button
- Fill in the required fields for the trial session
- Click Add Session
- Next, click on the New tab and then click on the City, State link for the session you just created
- Select a few cases from the Eligible case list, making sure that you select at least 1 case that has petitioners with paper service.
- Click Set Calendar
- Review the Print for paper service screen

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the Notice setting case for Trial for each case displays the CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".
- [x] **NEW FUNCTIONALITY** - After clicking done and opening some of the cases on the trial session, Click on the Notice that was just generated on the docket record - The Clerk of the Court signature at the bottom of the Notice setting case for Trial is displayed correctly, title is Clerk of the Court displays "Acting Clerk of the Court".



**Part 6 - Notice of Change to Remote Proceeding**

- Continuing from the previous Test
- Click on trial Sessions
- Click on the city, state link for the trail session from the previous test
- Click on the edit link
- Select the remote radio button, and fill in the required fields for a remote proceeding
- Save

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the Notice of Change to Remote Proceeding for each case displays the CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".
- [x] **NEW FUNCTIONALITY** - After clicking done and opening some of the cases on the trial session, Click on the Notice that was just generated on the docket record - The Clerk of the Court signature at the bottom of the Notice of remote proceeding is displayed correctly, title is Clerk of the Court displays "Acting Clerk of the Court".



**Part 7 - Notice of Change to In Person Proceeding**

- Continuing from the previous Test
- Click on trial Sessions
- Click on the city, state link for the trail session from the previous test
- Click on the edit link
- Select the In-Person radio button, and fill in the required fields for an In-Person proceeding
- Save

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the Notice of Change to In-Person Proceeding for each case displays the CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".
- [x] **NEW FUNCTIONALITY** - After clicking done and opening some of the cases on the trial session, Click on the Notice that was just generated on the docket record - The Clerk of the Court signature at the bottom of the Notice of In-Person Proceeding is displayed correctly, title is title is Clerk of the Court displays "Acting Clerk of the Court".



**Part 8 - Notice of Change of Trial Judge**

- Continuing from the previous Test
- Click on trial Sessions
- Click on the city, state link for the trail session from the previous test
- Click on the edit link
- Scroll down to the Judge Dropdown, and select a different Judge.  Be sure all required fields are filled out.
- Save

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the Notice of Change of Trial Judge for each case displays the CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".
- [x] **NEW FUNCTIONALITY** - After clicking done and opening some of the cases on the trial session, Click on the Notice that was just generated on the docket record - The Clerk of the Court signature at the bottom of the Notice of change of trial judge is displayed correctly, title is Clerk of the Court displays "Acting Clerk of the Court".



**Part 9 - 30 Day Notice of Trial**

- Log in as a Petitions Clerk (or CSS, or CotC)
- Click on Trial Sessions 
- Scroll down until you find a trial session that has a clock icon present
- Click on the City, state link for the trial session with a clock icon
- Click on the Serve/Dismiss link in the yellow alert banner
- Click Yes, Serve button in the pop-up.
- Review the paper service page

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the 30 Day Notice of Trial for each case displays the CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".
- [x] **NEW FUNCTIONALITY** - After clicking Printing Complete and opening some of the cases on the trial session, Click on the Notice that was just generated on the docket record - The Clerk of the Court signature at the bottom of the 30 Day Notice of Trial is displayed correctly, title is Clerk of the Court displays "Acting Clerk of the Court".



#### 3) DAWSON Technical Team member updates the Clerk of the Court's Name and Title; Notices in DAWSON Display the updated CotC Name and Title. 

- As a Technical Team Member, update the CotC name AND the title (put "acting Clerk of the Court") (Dev Determination on how to do this)

**Part 1 - NOTR/Notice of attachments - eFiled Petition**
- Next, Log into DAWSON as a Petitions Clerk
- Click on Document QC
- Find a Petition that is in New status (avoid the test petitions that are in a status other than new)
- Click on the Petition link
- Be sure that you click on the radio button for Notice of Attachments in the Nature of Evidence while you are QCing the eFiled Petition.
- Serve the Petition to the IRS
- Click on the Served tab
- Click on the docket number of the Petition that you just served.
- Click on the Notice of Receipt of Petition on the docket record

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court name at the bottom of the page displays the updated CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".

- Next, click on the Case information tab
- Click on the Print Confirmation link

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the page displays the updated CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".

- Next, click on the drafts tab and find the Notice of Attachments in the Nature of Evidence draft

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Draft Notice displays the updated CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".
- [x] **NEW FUNCTIONALITY** - After clicking on the Add Docket Entry link the document preview on the Add Docket Entry screen has the updated CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".
- [x] **NEW FUNCTIONALITY** - After Serving the document, the document on the docket record displays the updated CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".



**Part 2 - NOTR/Notice of attachments - Paper-Filed Petition**
- Log into DAWSON as a Petitions Clerk
- Click on Document QC
- Click on Start a Petition button
- Upload or scan in documents and fill in all the required fields
- Be sure that you click on the radio button for Notice of Attachments in the Nature of Evidence while you are QCing the eFiled Petition.
- Create the Case, and then Serve the Petition to the IRS

Expected Results
- [x] **NEW FUNCTIONALITY** - On the Print Paper Petition Receipt screen, The Clerk of the Court name at the bottom of the NOTR displays the updated CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".  
- [x] If the document is printed, the Petitioner address fits in the envelope window as per normal.

- Next, click the done button
- Click on the Served link in your Document QC
- Click on the docket number link of the petition you just served
- Click on the Notice of Receipt of Petition on the docket record

Expected Results:
- [x] Notice of receipt of petition (Check to be sure that the address window still looks good)
- [x] **NEW FUCTIONALITY** - The Clerk of the Court name at the bottom of the page displays the updated CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".

- Next, click on the Case information tab
- Click on the Print Confirmation link

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the page displays the CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".

- Next, click on the drafts tab and find the Notice of Attachments in the Nature of Evidence draft

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Draft Notice displays the updated CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".
- [x] **NEW FUNCTIONALITY** - After clicking on the Add Docket Entry link the document preview on the Add Docket Entry screen has the updated CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".
- [x] **NEW FUNCTIONALITY** - After Serving the document, the document on the docket record displays the updated CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".



**Part 3 - Generate a Notice on a case**

- Continuing from part 2 above
- Click on the Create button in the case
- Select Order or Notice
- Select Notice and then click Continue
- Type in some text in the Generate section on the Create Notice screen
- Click on the Preview tab

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the page displays the updated CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".

- Next, Click on the Save button

Expected Results:
- [x] **NEW FUNCTIONALITY** - The document preview on the Drafts page displays The Clerk of the Court signature at the bottom of the page displays the updated CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".

- Next, click on the Add docket Entry link

Expected Results:
- [x] **NEW FUNCTIONALITY** - The document preview on the Add Docket Entry page displays The Clerk of the Court signature at the bottom of the page displays the updated CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".

- Next, Save and Serve the document
- Review the Print Notice for Paper Service Screen

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the page displays the updated CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".



**Part 4 - Notice of Docket Change**

- Log in as a party that has electronic access to a case
- Click on the docket number for the case on the docket record
- Click on File a document button
- Select a document title, upload a document, and submit the document to the Court
- Next, log in as a docket clerk
- Click on Document QC, and then Section Document QC 
- Click on the document title link for the document that you just filed
- Change the document title and click Complete
- Navigate back to the case docket record
- Click on the automatically generated Notice of Docket Change

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the page displays the updated CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".



**Part 5 - Initial Calendaring**

- Log in as a Petitions clerk (or CotC, or CSS)
- Click on Trial Sessions
- Click on Add Trial Session button
- Fill in the required fields for the trial session
- Click Add Session
- Next, click on the New tab and then click on the City, State link for the session you just created
- Select a few cases from the Eligible case list, making sure that you select at least 1 case that has petitioners with paper service.
- Click Set Calendar
- Review the Print for paper service screen

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the Notice setting case for Trial for each case displays the updated CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".
- [x] **NEW FUNCTIONALITY** - After clicking done and opening some of the cases on the trial session, Click on the Notice that was just generated on the docket record - The updated Clerk of the Court signature at the bottom of the Notice setting case for Trial is displayed correctly, title is Clerk of the Court displays "Acting Clerk of the Court".



**Part 6 - Notice of Change to Remote Proceeding**

- Continuing from the previous Test
- Click on trial Sessions
- Click on the city, state link for the trail session from the previous test
- Click on the edit link
- Select the remote radio button, and fill in the required fields for a remote proceeding
- Save

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the Notice of Change to Remote Proceeding for each case displays the updated CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".
- [x] **NEW FUNCTIONALITY** - After clicking done and opening some of the cases on the trial session, Click on the Notice that was just generated on the docket record - The updated Clerk of the Court signature at the bottom of the Notice of remote proceeding is displayed correctly, title is Clerk of the Court displays "Acting Clerk of the Court".



**Part 7 - Notice of Change to In Person Proceeding**

- Continuing from the previous Test
- Click on trial Sessions
- Click on the city, state link for the trail session from the previous test
- Click on the edit link
- Select the In-Person radio button, and fill in the required fields for an In-Person proceeding
- Save

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the Notice of Change to In-Person Proceeding for each case displays the updated CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".
- [x] **NEW FUNCTIONALITY** - After clicking done and opening some of the cases on the trial session, Click on the Notice that was just generated on the docket record - The updated Clerk of the Court signature at the bottom of the Notice of In-Person Proceeding is displayed correctly, title is title is Clerk of the Court displays "Acting Clerk of the Court".



**Part 8 - Notice of Change of Trial Judge**

- Continuing from the previous Test
- Click on trial Sessions
- Click on the city, state link for the trail session from the previous test
- Click on the edit link
- Scroll down to the Judge Dropdown, and select a different Judge.  Be sure all required fields are filled out.
- Save

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the Notice of Change of Trial Judge for each case displays the updated CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".
- [x] **NEW FUNCTIONALITY** - After clicking done and opening some of the cases on the trial session, Click on the Notice that was just generated on the docket record - The updated Clerk of the Court signature at the bottom of the Notice of change of trial judge is displayed correctly, title is Clerk of the Court displays "Acting Clerk of the Court".



**Part 9 - 30 Day Notice of Trial**

- Log in as a Petitions Clerk (or CSS, or CotC)
- Click on Trial Sessions 
- Scroll down until you find a trial session that has a clock icon present
- Click on the City, state link for the trial session with a clock icon
- Click on the Serve/Dismiss link in the yellow alert banner
- Click Yes, Serve button in the pop-up.
- Review the paper service page

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Clerk of the Court signature at the bottom of the 30 Day Notice of Trial for each case displays the updated CotC name, title is Clerk of the Court displays "Acting Clerk of the Court".
- [x] **NEW FUNCTIONALITY** - After clicking Printing Complete and opening some of the cases on the trial session, Click on the Notice that was just generated on the docket record - The updated Clerk of the Court signature at the bottom of the 30 Day Notice of Trial is displayed correctly, title is Clerk of the Court displays "Acting Clerk of the Court".




FYI - Places that display the CotC signature in DAWSON:

- 30 Day Notice of Trial
- Notice of Change to Remote Proceeding
- Notice of Change to In-Person Proceeding
- Notice of Change of Trial Judge
- Notice of receipt of petition (Check to be sure that the address window still looks good)
- Notice of attachments in the nature of evidence
- Any Generic Notice created by the Notice Generator
- Notice of Docket Change
- Notice Setting Case for Trial

**Comment by ttlenard** [comment]
@codyseibert @Absolutestunna Testing looks good. Thank you!
